tests/e2e/tests/checkout/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const SIMPLE_VIRTUAL_PRODUCT_NAME = 'Album';
+export const SIMPLE_PHYSICAL_PRODUCT_NAME = 'Beanie';
+export const FREE_SHIPPING_NAME = 'Free shipping';
+export const FREE_SHIPPING_PRICE = '$0.00';
```

Comment:
If we're adding these constants can we update https://github.com/woocommerce/woocommerce-blocks/blob/trunk/tests/e2e/tests/checkout/checkout-block-shipping.block_theme.side_effects.spec.ts#L21-L24 so it uses them too?